cylc/flow/scripts/lint.py

```diff
@@ -28,15 +28,16 @@
 
 By default, suggestions are written to stdout.
 
-In-place mode ( "-i, --inplace") writes suggestions into the file as comments.
+In-place mode ("-i, --inplace") writes suggestions into the file as comments.
 Commit to version control before using this, in case you want to back out.
```

Comment:
Could do with mentioning you can use pyproject.toml in the help?

Reply:
I had been going to simply put it into the documentation in cylc-doc, but I agree.